command_server/command_server.py

```diff
@@ -1,12 +1,13 @@
 import asyncio
-import aiohttp
 import configparser
 import logging
+import aiohttp
 
-from aiohttp import server
+from aiohttp import server, web
 from aiohttp.multidict import MultiDict
-from routes import ROUTES
+from generic import routes
 from urllib.parse import urlparse, parse_qsl
+import views
 
 
 class HttpRequestHandler(aiohttp.server.ServerHttpProtocol):
```

Comment:
What is the main idea of HttpRequestHandler with web.Application ? Can you promise  that request will ne handled in one common method?